src/components/Button/Button.js

```diff
@@ -11,9 +11,13 @@ const { block } = bem({
 });
 
 const Button = props => {
-    const { children, context, disabled, isBlock, isInline, type, ...rest } = props;
+    const { children, context, disabled, isBlock, isInline, type, href, ...rest } = props;
 
-    return (
+    return href ? (
+        <a {...rest} {...block(props)} hfer={href}>
+            {children}
+        </a>
+    ) : (
```

Comment:
I think using a shorthand `if` is not very readable in this case. Better use two `return` statements and an explicit `if`.

```
if (href) {
   return (anchor);
}

return (button);
```